src/esp/sensor/CameraSensor.h

```diff
@@ -14,6 +14,7 @@ namespace sensor {
 
 struct CameraSensorSpec : public VisualSensorSpec {
   float orthoScale = 0.1f;
+  Mn::Color4 clearColor = {0, 0, 0, 1};
```

Comment:
```suggestion
  Mn::Color4 clearColor{0, 0, 0, 1};
```

Reply:
It's more consistent this way, haha :trollface: